src/switch_rtp.c

```diff
@@ -2050,6 +2057,78 @@ static int using_ice(switch_rtp_t *rtp_session)
 	return 0;
 }
 
+static void switch_send_rtcp_event(switch_rtp_t *rtp_session ,struct switch_rtcp_sender_report *sr,struct switch_rtcp_report_block *rtcp_report_block) {
```

Comment:
"{" - open the curly bracket after a new line, not on the line with the function name.

Reply:
Done :)